Sources/ArcGISToolkit/Documentation.docc/Tutorials/JobManagerTutorial.tutorial

```diff
@@ -0,0 +1,138 @@
+        @ContentAndMedia {
+            @Image(source: JobManagerTutorial, alt: "An image of the JobManager component.")
+            In this tutorial you will use a job that generates an offline map. The job will connect to a service, ask the service to generate an offline map, poll the status of the job, and then download the offline map once it is ready.
+        }
+        
+        @Steps {
+            @Step {
+                Create the JobManagerTutorialView struct. Add properties and start the UI for adding a job to the job manager. 
+                
+                Begin by adding a JobManager property as an observed object. Add state properties for `job`, `error`, and a boolean that you'll use to show progress when adding a job. Start the UI by adding a button to take a map offline and a progress indicator to show that the job is being added. This will be the UI before a job has been added (`job == nil`).
+                
+                @Code(name: "JobManagerTutorialView.swift", file: JobManagerViewStep1) {
+                    @Image(source: JobManagerTakeMapOfflineButton, alt: "An Image of an iPhone app with a Take Map Offline button.")
+                }
+            }
+            
+            @Step {
+                Add an extension containing the helper function for making the job.
+                
+                The function makes a map from a URL, then uses a task to make the generate offline map job.
+                
+                @Code(name: "JobManagerTutorialView.swift", file: JobManagerViewStep2, previousFile: JobManagerViewStep1)
+            }
+            
+            @Step {
+                Make the UI that will show when a job has been added to the job manager.
+                
+                To begin, add a state variable for the job status. Just after the check to see if we have a job, add a progress view that will show the progress of the job. Next, add a button to remove the job from the job manager and start over; give it the title "Start New Job". Last add a task to keep track of the job status in the status state variable.
+                
+                @Code(name: "JobManagerTutorialView.swift", file: JobManagerViewStep3, previousFile: JobManagerViewStep2) {
+                    @Image(source: JobManagerTutorial, alt: "An Image of an iPhone app with a progress bar and Start New Job button.")
+                }
+            }
+            
+            @Step {
+                Let's look at what we have so far. When the "Take Map Offline" button is tapped, a job is added to the job manager and the job is started.
+                
+                The first thing the job does is connect to the specified server to kick off the process that will make a mobile map package. Jobs are intended to be used with longer running service tasks. Once the task on the server is started the job waits and occasionally polls the service for the status of the service task.
```

Comment:
The term "Task" is a bit overloaded. We might want to avoid using it. The term "work" is used below, and that seems better.